src/data/features/schema.js

```diff
@@ -11,8 +11,16 @@ export default gql`
         READ_GROUP
     }
 
+    enum HORIZONTAL_CARD_TYPE {
```

Comment:
Aren't `enums` typically in `TitleCase` too? Then the values in `CAPS_SNAKE_CASE`?

```suggestion
    enum HorizontalCardType {
```

Reply:
Gotcha! Sounds good to me. I see both conventions in the Core Apollos project, but if this approach is more typical of GQL globally, let's for sure go with Upper Camel Case